Header header
bool goalpost_detected
int8 goalpost_number   # 3 if only see the middle bar of goalpost. then send out the left and right point as left goalpost and right goalpost
float64 goalpost_left_range
float64 goalpost_left_bearing
float64 goalpost_right_range
float64 goalpost_right_bearing
float64 goalpost_center_range
float64 goalpost_center_bearing
